admin/data-collection.rst

```diff
@@ -762,28 +754,25 @@ NetObj                   Object connected with this :term:`DCI`
 ======================== ==========================================================
 
 
-Performance tab
----------------
+.. _dci-performance-view:
 
-Main information about node(:guilabel:`Object Details`) can be supplemented with
-DCI information displayed as text(last value) on :guilabel:`Object Details`->
-:guilabel:`Overview` page or in graph way on :guilabel:`Object
-Details`->:guilabel:`Performance` tab.
+Performance view
+----------------
+
+This section provides configuration options for displaying DCI values as line charts on the :guilabel:`Performance tab`. Various options are available to visually represent the collected data; see :ref:`Data and Network visualization <visualisation>` for more details.
 
-DCI representation in text way can be configured on
-:ref:`dci-othe-options-label`. Next will be described only graph DCI
-representation configuration on :guilabel:`Performance` tab of :guilabel:`Object
-Details`.
+.. note::
 
-Multiple DCIs can be grouped in one graph. To group them use the same group name
-in "Group" field.
+  Note: Not available for table metrics.
 
 
-.. figure:: _images/dci_performance_tab_page.png
```

Comment:
Please rename _images/dci_perf_view_page.png to _images/dci_performance_tab_page.png and use that name here. 

Our standard is not to shorten words if it is possible.